modules/monitoring-granting-the-viewing-and-editing-roles-to-a-user.adoc

```diff
@@ -2,23 +2,22 @@
 //
 // * monitoring/monitoring-your-own-services.adoc
 
-[id="granting-the-role-to-a-user_{context}"]
-= Granting the role to a user
+[id="granting-the-viewing-and-editing-roles-to-a-user_{context}"]
+= Granting the viewing and editing roles to a user
 
-This procedure shows how to assign the `monitor-crd-edit` role to a user.
+This procedure shows how to assign the `monitoring-rules-view`, `monitoring-rules-edit`, and `monitoring-edit` roles to a user.
```

Comment:
We should mention why they want this. e.g. this is meant for creating alerting and recording rules...